miranda/structure/_structure.py

```diff
@@ -78,89 +78,59 @@ def build_path_from_schema(
-                / facets["activity"]
-                / facets["source"]
-                / facets["project"]
-                / facets["domain"]
-                / facets["frequency"]
-                / facets["variable"]
+            folder_tree_structure = (
+                "type",
+                "institution",
+                "source",
+                "domain",
+                "frequency",
+                "variable",
             )
 
         if facets["type"] == "simulation":
             SIMULATION_SCHEMA.validate(facets)
-            if facets["processing_level"] == "raw":
-                try:
-                    if facets["project"] == "CORDEX":
-                        return (
-                            Path(output_folder)
-                            / facets["type"]
-                            / facets["processing_level"]
-                            / facets["activity"]
-                            / facets["mip_era"]
-                            / facets["project"]
-                            / facets["domain"]
-                            / facets["source"]
-                            / facets["driving_model"]
-                            / facets["experiment"]
-                            / facets["member"]
-                            / facets["frequency"]
-                            / facets["variable"]
-                        )
-                except KeyError:
-                    return (
-                        Path(output_folder)
-                        / facets["type"]
-                        / facets["processing_level"]
-                        / facets["activity"]
-                        / facets["mip_era"]
-                        / facets["domain"]
-                        / facets["institution"]
-                        / facets["source"]
-                        / facets["experiment"]
-                        / facets["member"]
-                        / facets["frequency"]
-                        / facets["variable"]
-                    )
-            elif facets["processing_level"] == "biasadjusted":
-                return (
-                    Path(output_folder)
-                    / facets["type"]
-                    / facets["processing_level"]
-                    / facets["activity"]
-                    / facets["mip_era"]
-                    / facets["bias_adjust_institution"]
-                    / facets["bias_adjust_project"]
-                    / facets["domain"]
-                    / facets["institution"]
-                    / facets["source"]
-                    / facets["experiment"]
-                    / facets["member"]
-                    / facets["frequency"]
-                    / facets["variable"]
-                )
-        raise ValueError()
+            folder_tree_structure = (
+                "type",
+                "processing_level",
+                "mip_era",
+                "activity"
+                if facets["processing_level"] == "raw"
```

Comment:
are you missing a `None` here ?

Reply:
This statement was broken into two lines:
```
                if facets["processing_level"] == "raw"
                else "bias_adjust_project"
```

Reply:
ah yes my bad!